.github/workflows/pr-comment.yaml

```diff
@@ -0,0 +1,24 @@
+name: PR comment
+on: issue_comment
+
+jobs:
+  pr_commented:
+    name: PR comment workflow
+    if: |
+      github.event.issue.pull_request &&
+      github.repository == 'adobe/react-spectrum' &&
+      github.event.comment.author_association == 'MEMBER'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
+      - name: Use Node 16
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - name: install
+        run: yarn install
+      - name: Comment contains trigger
+        if: contains(github.event.comment.body, 'get build link')
```

Comment:
is this comment too generic? Maybe something that might be accidentally typed? Perhaps it should be `GET_BUILD` or something hmm